ci/redis/Dockerfile

```diff
@@ -0,0 +1,3 @@
+FROM redis:alpine
```

Comment:
Maybe, it can be useful for debugging potential issues / investigate security vulnerabilities to pin the redis / alpine version so that it is immediately evident, e.g. `6.2.0-alpine-3.13`

Reply:
agreed!